src/screens/FeedScreen/elements/Search.js

```diff
@@ -98,7 +98,7 @@ export const HeaderStyles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holytosca,
+    color: COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, this change is good for maintainability and consistency. Ensure `COLORS.anon_primary` is defined and has the same color value as `COLORS.holytosca`.
```diff
-    color: COLORS.holytosca,
+    color: COLORS.anon_primary,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->